### PR TITLE
[Draft] edited colors formats in styled components

### PR DESCRIPTION
**Pull request by Dekshut**
- [ ] **Breaking change?** (if so, please describe the impact and migration path for existing applications:)

**What changes did you make?** (Give an overview)
Edited using colors in styled components by our coloros Theme
**Is there anything you'd like reviewers to focus on?**
Added new grey object to Theme

**How Has This Been Tested?** (put an "X" next to an item)

- [ ] No need to
- [X] Manually(please, describe, when necessary)
- [ ] Unit checks
- [ ] Integration checks
- [ ] Covered by existing automation

**Checklist** (put an "X" next to an item, otherwise PR will fail)
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation (e.g. **ENVIRONMENT VARIABLES**)
- [X] My changes generate no new warnings(e.g. Sonar is happy)
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes
- [X] Any dependent changes have been merged

Check out [Contributing](https://github.com/provectus/kafka-ui/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/provectus/kafka-ui/blob/master/CODE-OF-CONDUCT.md)

### CODE REVIEW

**Review by Motherships — COMMENTED** [review]
Using color directly from `Color` object is undesired approach  
@workshur Suggested to use `theme` instead and add properties to `theme` if needed

File: kafka-ui-react-app/src/theme/theme.ts [hunk 1]
```diff
@@ -40,6 +41,13 @@ export const Colors = {
     '30': '#90caf9',
     '40': '#64b5f6',
   },
+  gray: {
+    '10': '#f9fafa',
+    '20': '#ebebeb',
+    '30': '#e7e7e7',
+    '40': '#e6e6e6',
+    '80': '#919497',
+  },
```
Comment (NelyDavtyan) [inline kafka-ui-react-app/src/theme/theme.ts:50]: Do you take these colors from figma? 
Oleg told me that we should take the colors from figma.

Comment (Dekshut) [reply kafka-ui-react-app/src/theme/theme.ts:50]: i got these colors from their rgba versions for add they to Theme

File: kafka-ui-react-app/src/components/Alerts/Alert.styled.ts [hunk 2]
```diff
@@ -10,7 +11,7 @@ export const Alert = styled.div<{ $type: AlertType }>`
   display: flex;
   justify-content: space-between;
   align-items: center;
-  filter: drop-shadow(0px 4px 16px rgba(0, 0, 0, 0.1));
+  filter: drop-shadow(0px 4px 16px ${Colors.gray[40]});
```
Comment (abelgibayev) [inline kafka-ui-react-app/src/components/Alerts/Alert.styled.ts:14]: Wrong approach. Do not import Colors, use theme.

**Pull request closed without merging**